### PR TITLE
add reference for keywordable entity

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] stock reference keyword

https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1734
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/232
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/583
https://github.com/open-orchestra/open-orchestra-model-interface/pull/188
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/31
https://github.com/open-orchestra/open-orchestra-libs/pull/51
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/219
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/201
https://github.com/open-orchestra/open-orchestra-elastica-bundle/pull/35

### CODE REVIEW

File: MediaModelBundle/Repository/MediaRepository.php [hunk 1]
```diff
@@ -51,7 +51,12 @@ public function findByKeywords($keywords)
     {
         $qb = $this->createQueryBuilder();
 
-        $qb->field('keywords.label')->in(explode(',', $keywords));
+        $keywords = explode(',', $keywords);
+        foreach($keywords as $key => $keyword) {
+            $keywords[$key] = new \MongoId($keyword);
```
Comment (itkg-ngilain) [inline MediaModelBundle/Repository/MediaRepository.php]: I'm not sure about this: if we want to get the mongo ids of existing keyword documents, we should then search for them in the database, not regenerate new ones ?

Comment (itkg-nanne) [inline MediaModelBundle/Repository/MediaRepository.php]: @itkg-ngilain $keyword is the id and new MongoId($keyword) is just a structure to query the database.

Comment (itkg-ngilain) [inline MediaModelBundle/Repository/MediaRepository.php]: @itkg-nanne : woops my bad, ok then

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**